src/pages/using-spark/guides/content-style.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Content Style
+---
+
+import { SprkDivider } from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Content Style
+</h1>
+
+UX Writers, designers and developers can use this **Content Style Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy for all Rocket Products digital experiences.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+## Main Takeaways
+
+- Use headings to label sections in a page according to their importance.
+- As a general rule, use **start case** for headers.
+- Use **sentence case** for body copy, email subject lines and pre-headers.
+- Use **double quotes** when quoting someone’s words.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Table Of Contents
+
+<ul>
+  <li>
+    <a href="#capitalization">Capitalization</a>
+  </li>
+  <ul class="sprk-b-List sprk-b-List--indented sprk-u-Measure sprk-u-mvs">
+    <li>
+      <a href="#start-case">Start Case</a>
+    </li>
+    <li>
+      <a href="#sentence-case">Sentence Case</a>
+    </li>
+  </ul>
+  <li>
+    <a href="#quotation-marks">Quotation Marks</a>
+  </li>
+  <li>
+    <a href="#accessibility">Accessibility</a>
+  </li>
+</ul>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+## Capitalization
+
+Use for visual presentation of pieces of text to make it easy for users to read.
+
+> As a general rule, use **start case** for **headers**, use either **start case** or **sentence case** for **dropdown options** and **labels**, depending on the context.
+
+### Start Case
+
+Capitalize the first letter of each word, regardless of length or part of speech. When using start case, avoid using end punctuation other than question marks.
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Usage</h3>
+
+- Headers that are descriptive or functional
+- All subheads regardless of whether the primary header is title or sentence case
+- All buttons labels (Primary, Secondary and Tertiary Buttons)
+- Menu items
+- Chart and table headers
+- Referring to the name of a page
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Examples</h3>
+
+- Homeowners Insurance Info
+- How To Buy A Home
+- Lock Your Rate And Save
+- Check Your To Do List
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+### Sentence Case
+
+Sentence case capitalizes the first word of the sentence and proper nouns.
```

Comment:
Sentence case capitalizes the first word of each sentence as well as proper nouns.